index.html

```diff
@@ -12,6 +12,17 @@
 </head>
 
 <body>
+    <header>
+
```

Comment:
It is good design wise to make the nav area the same size as the rest of the page since that is what the design file wants you to do.